### ISSUE TITLE
NPM is out of date

### ISSUE THREAD

**Issue by PauloGoncalvesBH** [vlucas/frisby#553]
Why is NPM out of date?
The most recent publication is 9 months ago

**Comment by LarryBattleWork** [comment]
What version should it be?
The master branch from May 15, 2020 gives a version of `2.1.2`.

https://github.com/vlucas/frisby/blob/master/package.json#L3

Which is the same version in NPM.

```
$ npm view frisby

frisby@2.1.2 | BSD-3-Clause | deps: 5 | versions: 81
Frisby.js v2.0: REST API Endpoint Testing built on Jasmine
http://frisbyjs.com

keywords: testing, api, REST, jasmine, bdd, frisby

dist
.tarball: https://registry.npmjs.org/frisby/-/frisby-2.1.2.tgz
.shasum: 14d3b5e5ffaa7c49ec909f0941ff80905e20fb5d
.integrity: sha512-+XOmw0J0RG0qWspkF9fuuCano9KYnhZp9m0YTcQjx75K3yBZRqLaaqoVRlexaHlPh+QdowjeDvsBi4uDg0ILJA==
.unpackedSize: 72.8 kB
```

What version should it be?

**Comment by PauloGoncalvesBH** [comment]
Since the release of version 2.1.2, several changes have been made, including 1 feature.

![image](https://user-images.githubusercontent.com/29241659/82695202-03dc6380-9c3b-11ea-9b1f-f78574058456.png)

**Comment by vlucas** [comment]
Just released v2.1.3 last night.
